tidb-cloud/cli-reference.md

```diff
@@ -17,17 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
-| cluster    | create, delete, describe, list, connect-info               | Manage clusters                                                                                          |
-| branch     | create, delete, describe, list, connect-info               | Manage branches                                                                                          |
-| completion | bash, fish, powershell, zsh                                | Generate completion script for specified shell                                                           |
-| config     | create, delete, describe, edit, list, set, use             | Configure user profiles                                                                                  |
-| connect    | -                                                          | Connect to a TiDB cluster                                                                                |
-| help       | cluster, completion, config, help, import, project, update | View help for any command                                                                                |
-| import     | cancel, describe, list, start                              | Manage [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
-| project    | list                                                       | Manage projects                                                                                          |
-| update     | -                                                          | Update the CLI to the latest version                                                                     |
+| Command            | Subcommand                                                               | Description                                                                                              |
+|--------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| auth               | login, logout                                                            | Login and logout                                                                                         |
+| serverless         | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |
+| serverless branch  | create, delete, describe, list, shell                                    | Manage TiDB Serverless branches                                                                          |
+| serverless import  | cancel, describe, list, start                                            | Manage TiDB Serverless [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
```

Comment:
why this link? I think it's not suitable here.

Reply:
it used to be this link, i will remove the link now